pyscriptjs/src/main.ts

```diff
@@ -285,24 +306,32 @@ export class PyScriptApp {
 
         // inject `define_custom_element` and showWarning it into the PyScript
         // module scope
-        // eventually replace the setHandler calls with interpreter._remote.setHandler i.e. the ones mentioned below
-        // await interpreter._remote.setHandler('define_custom_element', Synclink.proxy(define_custom_element));
-        // await interpreter._remote.setHandler('showWarning', Synclink.proxy(showWarning));
-        interpreter._unwrapped_remote.setHandler('define_custom_element', define_custom_element);
-        interpreter._unwrapped_remote.setHandler('showWarning', showWarning);
+        if (interpreter.useWorker) {
+            // IMPLEMENT ME!
+            // await interpreter._remote.setHandler('define_custom_element', Synclink.proxy(define_custom_element));
+            // await interpreter._remote.setHandler('showWarning', Synclink.proxy(showWarning));
```

Comment:
`showWarning` we can do pretty easily with same thread syncify. For `define_custom_element` I'd have to look a bit more carefully at the API to get a clear concept.

Reply:
See #1385.